### ISSUE TITLE
Re-examine need for provider details

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#564]
Not all providers may includeitem-level information. ReShare partners will, but it is likely that POD may expand beyond the ReShare partners, and include partners that do not share item-level information. 

I would like a field to place a URL that will link to a local page describing our info in detail: 
https://wiki.harvard.edu/confluence/display/LibraryStaffDoc/POD+Aggregator+%28IvyPlus%29+data+set+and+mapping

I find the other fields to be problematic. AFAIK POD does not need them. Data provider requirements specificy that local bibliographic identifier be in a MARC 001 field so we don't want to suggest there are other options for that. As written it sounds like the field might be intended for local ITEM identifier, which is different, and I don't know why we'd want to define that here. Call numbers can span multiple subfields so I would not leave a field here for that. Same for local library. In short I think it's simplest if orgs can just include a link to where they define their data local data elements.

_Originally posted by @cbaksik in https://github.com/pod4lib/aggregator/issues/542#issuecomment-1092197109_

**Comment by JohnMarkOckerbloom** [comment]
I agree with @corylown that we should add a field here for linking to information on each organization's encoding of things like holdings, items, lendability, and inclusion in BorrowDirect.  That field should appear first on this page.  I'm not sure what the best label for this field is, but we could always give it a short name and say more about it in the org owner's manual.

James is collecting that information now from Ivy Plus partners. If an organization has such a document, but does not have a stable home for it, we could consider hosting it on our documentation Wiki and pointing to that copy, though such files might not be easily updatable by providers.  (Alternatively, we could have some sort of file upload and storage service on this page for such files, but I'm not sure that capability is worth the effort of implementation at this point.)

Cory mentioned at today's workcycle planning meeting that the fields presently on this page do get used by our record normalization scripts.  It might be good to verify which ones are, or need to be, and remove ones that aren't necessary or used.  

Many organizations do not fill in these fields, or do not have accurate or up to date information in them.  (Penn is one example of the latter at the moment.)  If we retain them, it might be good to have a short note, possibly linking to more details in the org owner's manual, about what theses fields are used for.

**Comment by marlo-longley** [comment]
@corylown does this PR cover this ticket?

https://github.com/pod4lib/aggregator/pull/835
